internal/cmdrender/runtime/container.go

```diff
@@ -41,6 +45,25 @@ type ContainerFnPermission struct {
 	AllowNetwork bool
 }
 
+// ContainerFnWrapper wraps the real function filter, prints
+// the function running progress and failures.
+type ContainerFnWrapper struct {
+	Fn *ContainerFn
+}
+
+func (fw *ContainerFnWrapper) Run(r io.Reader, w io.Writer) error {
+	pr := printer.FromContextOrDie(fw.Fn.Ctx)
+	printOpt := printer.NewOpt().Indent(printer.FnIndentation)
```

Comment:
This feels a bit backwards. Why does the client need to specify the indentation (which is completely presentation-related) when it is just taken from a constant within the printer package?

Reply:
Right. This indent can be removed.

Reply:
done